js/apps/account-ui/test/oid4vci/oid4vci.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { expect, test } from "@playwright/test";
+import { login } from "../login";
+
+const realm = "verifiable-credentials";
+
+test.describe("Verifiable Credentials page", () => {
+  test("Get offer for test-credential.", async ({ page }) => {
+    await login(page, "test-user", "test", realm);
+
+    await expect(page.getByTestId("qr-code")).toBeHidden();
+
+    await page.getByTestId("oid4vci").click();
+    await page.getByTestId("credential-select").click();
+
+    await expect(
+      page.getByTestId("select-verifiable-credential"),
+    ).toBeVisible(),
```

Comment:
Looks like some code formatting issues here, make sure you have the linter enabled. If you are using VSCode I can recommend the [ESLint plugin](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint). There are also options for [JetBrains stuff](https://www.jetbrains.com/help/idea/eslint.html)

Reply:
Thats what the linter in the pipeline enforced(e.g. pnpm --fail-if-no-match --filter @keycloak/keycloak-admin-client lint). Looks weird to me to,  but I thought its on intention:)

Reply:
The comma should be a semicolon here.

Reply:
Fixed in [14a182e](https://github.com/keycloak/keycloak/pull/30127/commits/14a182ede2f779656143ce3eb229efb3af47b5ba)